app/_includes/service-catalog-integrations.html

```diff
@@ -7,6 +7,9 @@
         <span style="font-size: 22px;display:block;margin-bottom:8px;">{{ i.name }}</span>
       </div>
       <div class="install-description">{{i.description}}</div>
+      {% if i.events == true %}
+      <span class="badge events"></span>
+      {% endif %}
```

Comment:
```suggestion
      <p>
      {% if i.events == true %}
      <br>
      <span class="badge events"></span>
      {% endif %}
```
Let's at least add a linebreak here, this is currently squished up against the description.